experimental/PropertyDDS/packages/property-changeset/src/test/squashReversible.spec.ts

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * @fileoverview In this file, we will test the squashing of two
+ *    reversible changesets.
+ */
+ import { expect } from 'chai';
+import { ChangeSet, SerializedChangeSet } from "../changeset";
+
+describe("Squash Reversible ChangeSets", function() {
+    it("Squashing artificial reversible changesets ", () => {
+        const cs1s: SerializedChangeSet = {
+            String: {
```

Comment:
Which scenario or set of call on the API produces these kind of changesets? 
Which type of operation ("modify" or insert")  is it ? it's currently missing in the test.  

I tried this and it works without the fix: 
 ```
 const cs1s: SerializedChangeSet = {
        modify: {
            String: {
                x: {
                    oldValue: "a",
                    value: "b",
                },
            },
         },
        };
        const cs2s: SerializedChangeSet = {
        modify: {
            String: {
                y: {
                    oldValue: "c",
                    value: "d",
                },
            },
          },
        };
```

Reply:
@nedalhy  This PM is only followup for #10766 which was implemented by our colleague @pswillcock who is currently on leave so I wanted to check with you what you think about it. I was trying the test without the fix but it did not work, I will have to check it again. I also do not believe that such ChangeSet without modifier (modify, insert, delete) can happen so this looks to be unneeded but anyway I wanted your confirmation. Please, check the discussion at #10766, I only wanted to bring this back on the table as nobody touched it for some time (your comment in that ticket was : It would be great if you can introduce a simple reproducible test in the PR as @ruiterr requested, then we can merge it.)